tests/request.spec.coffee

```diff
@@ -286,3 +253,49 @@ describe 'Request:', ->
 							foo: 'bar'
 					.get('body')
 					m.chai.expect(promise).to.eventually.become(foo: 'bar')
+
+		describe 'given an endpoint that fails the first two times', ->
+
+			beforeEach ->
+				requestsSeen = 0
+				fetchMock.get 'https://example.com/initially-failing', ->
```

Comment:
Definitely better, done.

I've done this as tiny patch I've pushed straight to master, referencing this issue. I assume that's the right approach for small tweaks like this when the PRs already merged - let me know there's a different process I should be following.